runtime/integration-test/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+pallet-xcm = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+polkadot-parachain = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+polkadot-primitives = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+polkadot-runtime-parachains = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+xcm = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+xcm-builder = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+
+# Cumulus
+cumulus-primitives-core = { git = "https://github.com/mangata-finance/cumulus", branch = "mangata-dev" }
+parachain-info = { git = "https://github.com/mangata-finance/cumulus", branch = "mangata-dev" }
+
+# Open-Runtime-Module-Library Dependencies
+orml-tokens = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-traits = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-xtokens = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-xcm-support = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-unknown-tokens = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-xcm = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+orml-asset-registry = { version = "0.4.1-dev", git = "https://github.com/mangata-finance/open-runtime-module-library", branch = "mangata-dev" }
+
+# Local
+mangata-kusama-runtime = { path = "../mangata-kusama", optional = true }
+
+pallet-xyk = { path = '../../pallets/xyk', version = '0.1.0' }
+pallet-bootstrap = { path = '../../pallets/bootstrap', version = '0.1.0' }
+xyk-runtime-api = { path = '../../pallets/xyk/runtime-api', version = '2.0.0' }
+pallet-sudo-origin = { path = '../../pallets/sudo-origin' }
+pallet-issuance = { path = '../../pallets/issuance' }
+pallet-multipurpose-liquidity = { path = '../../pallets/multipurpose-liquidity' }
+mp-multipurpose-liquidity = { path = '../../primitives/multipurpose-liquidity' }
+mp-traits = { path = '../../primitives/traits' }
+mp-bootstrap = { path = '../../primitives/bootstrap' }
+
+[dev-dependencies]
+env_logger = "0.9.1"
+log = "0.4.17"
+sp-io = { git = "https://github.com/mangata-finance/substrate", branch = "mangata-dev" }
+xcm-executor = { git = "https://github.com/mangata-finance/polkadot", branch = "mangata-dev" }
+xcm-emulator = { git = "https://github.com/shaunxw/xcm-simulator", rev = "6c02911a3eeb04c342e4eb273d2ffbedd45dabff" }
+
+[features]
+default = ["with-kusama-runtime"]
+with-kusama-runtime = [
+    "mangata-kusama-runtime",
+]
```

Comment:
there are more deps than needed at this time but the plan is to add more tests for eg. proxy, xcm transfer etc...